server/datastore/mysql/migrations/tables/20240925111236_AddInstallDuringSetupToSoftwareInstallers.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240925111236, Down_20240925111236)
+}
+
+func Up_20240925111236(tx *sql.Tx) error {
+	_, err := tx.Exec(`ALTER TABLE software_installers ADD COLUMN install_during_setup BOOL NOT NULL DEFAULT false`)
+	if err != nil {
+		return fmt.Errorf("failed to add install_during_setup to software_installers: %w", err)
+	}
+
+	_, err = tx.Exec(`ALTER TABLE vpp_apps_teams ADD COLUMN install_during_setup BOOL NOT NULL DEFAULT false`)
+	if err != nil {
+		return fmt.Errorf("failed to add install_during_setup to vpp_apps_teams: %w", err)
+	}
+
+	_, err = tx.Exec(`ALTER TABLE host_script_results ADD install_during_setup BOOL NOT NULL DEFAULT false`)
```

Comment:
It might be worth adding a foreign key column to host_script_results pointing to the new setup experience scripts table in the other file instead of a bool

Reply:
The upside is that it's clear which setup script it belongs to without searching the other table for the row that references this row. The downside is that once the script is deleted and we set the foreign key in the `host_script_results` table to `NULL`, we no longer know that the script was run by a setup script, and it now just looks like an ad-hoc script run

Reply:
@roperzh Going with foreign key for now, would appreciate any opinions on this